src/main/java/org/zaproxy/zap/extension/hud/tutorial/pages/IndexPage.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
```

Comment:
We're in `2019` now :+1:

Reply:
Ha! Remembered to change that in the other PR, but not this one :P 
Will fix...

Reply:
No biggy, I'm still making that mistake too.

Reply:
Fixed :)